docs/notes.rst

```diff
@@ -1,6 +1,42 @@
-Version 1.0.0
+Version 1.0.2
 =============
 
+**Agent 1.0.2 | Client 1.0.1**
```

Comment:
It could be cool if we put a "compatibility table" that relates which the agent and client versions are compatibles between them.